src/application/CMakeLists.txt

```diff
@@ -13,7 +21,14 @@ elseif (UNIX)
   set_target_properties(f3d PROPERTIES INSTALL_RPATH $ORIGIN/../lib)
 endif ()
 
-target_compile_features(f3d PRIVATE cxx_std_11)
+# Cannot upgrade to C++17 here, because <codecvt> header is deprecated and produces warnings on Windows
```

Comment:
where do we use this and will it causes issues in the future ?

Reply:
It's deprecated but it will not be removed from the standard until there is an alternative. We can upgrade to C++17 and remove the deprecation warning in the future.